### PR TITLE
remove the plane segment part according to the new algorithm

### PR DESCRIPTION
**Pull request by ahuizxc**

### CODE REVIEW

File: object_analytics_node/src/segmenter/organized_multi_plane_segmenter.cpp [hunk 1]
```diff
@@ -94,19 +86,17 @@ void OrganizedMultiPlaneSegmenter::segmentPlanes(
...
   double start = pcl::getTime();
-
-  std::vector<bool> plane_labels;
-  plane_labels.resize(label_indices.size(), false);
-  for (size_t i = 0; i < label_indices.size(); i++) {
-    if (label_indices[i].indices.size() > plane_minimum_points_) {
-      plane_labels[i] = true;
-    }
+  PointCloud<Label>::Ptr labels(new PointCloud<Label>);
+  for (size_t i = 0; i<cloud->size(); i++){
```
Comment (yechun1) [inline object_analytics_node/src/segmenter/organized_multi_plane_segmenter.cpp]: please add blank beside ```<```, please run ```ament test``` to check the code style

**Comment by ahuizxc** [comment]
already passed the test :)

**Comment by ahuizxc** [comment]
有一个代码审查工具建议加上空格。。那我去掉了

**Comment by yechun1** [comment]
lgtm, thanks @ahuizxc

**Pull request merged**